src/components/search-result-list/search-result-list-item/search-result-list-item.tsx

```diff
@@ -98,7 +98,7 @@ const SearchResultListItem: React.FC<SearchResultListItemProps> = ({
     <article
       className="search-result-item arrow arrow__hover--right-small"
       onClick={handleClick}
-      onKeyUp={handleClick}
+      onKeyUp={(e) => e.key === "Enter" && handleClick}
```

Comment:
Consider allowing both Enter and Space as would be the case if the element was a `<button>`.